src/popup/index.css

```diff
@@ -0,0 +1,17 @@
+@tailwind base;
```

Comment:
are we using tailwind for this?

Reply:
yeah, it is compiled by the same bundler that does the devtools pane